scripts/results/arrange-data.py

```diff
@@ -144,66 +202,125 @@ def check_file(name):
-    file_name = d['i'][0]
-    with open(file_name) as f:
-        content = f.readlines()
+result = parser.parse_args(sys.argv[1:])
+   
+d = vars(result)
+file_name = d['i'][0]
+with open(file_name) as f:
+    content = f.readlines()
     content = [x.strip() for x in content]
     process()
     print 'Number of video sessions: ' + str(session_num)
+    start_times = rc[0][0].split()[1] + '/' \
+        + rc[0][0].split()[2] + '/' + rc[0][0].split()[4] + ' ' \
+        + rc[0][0].split()[3]
     print 'Earliest time is ' + rc[0][0].split()[1] + '/' \
         + rc[0][0].split()[2] + '/' + rc[0][0].split()[4] + ' ' \
         + rc[0][0].split()[3]
-
+    end_times = rc[-1][0].split()[1] + '/' \
+        + rc[-1][0].split()[2] + '/' + rc[-1][0].split()[4] + ' ' \
+        + rc[-1][0].split()[3]
     print 'Lastest time is ' + rc[-1][0].split()[1] + '/' \
         + rc[-1][0].split()[2] + '/' + rc[-1][0].split()[4] + ' ' \
         + rc[-1][0].split()[3]
-    print 'Please type a date range'
-    date_range = sys.stdin.readline()
-    type_plot = d['p'][0]
-    command = ''
-    if type_plot == 'jitter' or type_plot == 'rtt':
-        command = 'python session-avg-' + type_plot + '.py'
-    else:
-        command = 'python session-' + type_plot + '.py'
-    judge = False
-    if len(date_range.strip()) == 0:
-        print 'print all plot'
-        judge = True
-        command = command + ' ' + file_name
-    elif date_range.strip().find('-') == -1:
-        print 'one date specific'
-        check_one_date(convert_date(date_range))
-        write_file()
-        command = command + ' qualified_data.txt'
-    else:
-        print 'two date specific'
-        check_two_date(convert_date(date_range.strip().split('-')[0]),
-                       convert_date(date_range.strip().split('-')[1]))
-        write_file()
-        command = command + ' qualified_data.txt'
-    process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE)
-    output, err = process.communicate()
-    print output
-    if judge == False:
-        os.remove('qualified_data.txt')
-except ValueError:
-    print 'Your input date is invalid; hour should between 0-23, minute and '+\
-    'second should between 0-59, day should be 1-30 or 31 '
-except IndexError:
-    print 'Your input is missing some part and correct date should' + \
-    'be month/day/year hour:minute:second'
-except KeyError:
-    print 'Use valid months please such as Nov,Dec'
+    readline.set_completer_delims('\t\n')
+    readline.parse_and_bind("set show-all-if-unmodified on")
+    readline.parse_and_bind("set completion-query-items 10000")
+    readline.parse_and_bind("tab: complete")
+    auto_complete = sorted(auto_complete, key=trunc_datetime2)
+    from_argu = True
+
+while (True):
+    try:
+        start_time = ''
+        end_time = ''
+        if result.p ==None:
+           print 'You can choose one of the network charaistics from jitter, rebuffers, rtt, retx, timeout, nack, segments'

```

Comment:
instead of manually writing the options use `network_options` and print out its content to print this message. This way if we chage `network_options` in future, fewer changes we will need to make.